Chapter_2/dockerfiles/README.md

```diff
@@ -1,3 +1,5 @@
 ### Docker based environment for Data Science
 
-- `Dockerfile` along with readme is located [here](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/dockerfiles/jupyter-notebook)
+* [jupyter-notebook](https:/`/github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/dockerfiles/jupyter-notebook) - <to be filled>
+
+- [cpu-jupyter-notebook](https://github.com/PacktPublishing/Production-Ready-Applied-Deep-Learning/tree/main/Chapter_2/dockerfiles/cpu-jupyter-notebook) - <to be filled>
```

Comment:
@leninworld please fill in each of them, also how are they different?

Reply:
cleaned up. CPU should not be there (it was something left out while I was working on)